androidComposePrototype/src/main/java/com/genesys/cloud/messenger/androidcomposeprototype/ui/testbed/Command.kt

```diff
@@ -12,7 +12,7 @@ enum class Command(val description: String) {
     REFRESH("refreshAttachment <attachmentId>"),
     FILE_ATTACHMENT_PROFILE("fileAttachmentProfile"),
     DEPLOYMENT("deployment"),
-    DETACH("detach"),
+    DETACH("detach<attachmentId"),
```

Comment:
lets have "space" between detach and <attachmentId>, aslo there is no `>` enclosing bracket for attId.

Reply:
Ok! Fixed!